<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>TableView</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>PlainTextEdit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionModelData"/>
   <addaction name="separator"/>
   <addaction name="actionAppend"/>
   <addaction name="actionInsert"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlighCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionFontBold"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>另存文件</string>
   </property>
   <property name="toolTip">
    <string>表格内容另存为文件</string>
   </property>
  </action>
  <action name="actionAppend">
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="toolTip">
    <string>添加一行</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除当前行</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionModelData">
   <property name="text">
    <string>数据预览</string>
   </property>
   <property name="toolTip">
    <string>模型数据显示到文本框里</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>文字左对齐</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>文字右对齐</string>
   </property>
  </action>
  <action name="actionAlighCenter">
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>文字居中对齐</string>
   </property>
  </action>
  <action name="actionFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体文字</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
